src/packages/client/package.json

```
@@ -113,6 +113,7 @@
     "rollup": "2.32.1",
     "rollup-plugin-dts": "1.4.13",
     "set-value": "3.0.2",
+    "source-map-support": "^0.5.19",
```

```suggestion
    "source-map-support": "0.5.19",
```
Because renovate will create a PR to do that if not 😄